### ISSUE TITLE
Tag showing as not existing in namespace even though generated g.xaml has no issues

### ISSUE THREAD

**Issue by danhol86** [AmmyUI/AmmyUI#121]
Hi.

Am trying to convert my existing code to Ammy but getting stuck loading my viewmodellocator as resource using following code however the resources section is throwing error saying ViewModelLocator 
 does not exist in my namespace:

`using AA.Plugins.Test.ViewModel

UserControl {
  Resources: [
	ViewModelLocator Key="PluginLocator" {}
  ]

  DataContext: Binding {
    Source: "{StaticResource PluginLocator}"
    Path: "Home"
  }

  TextBox {
    Text: Binding {
      Path: "Test123"
    }

  }
}
`

The converted code compiles fine and even works when I run tha app:

`<UserControl xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:ns0="clr-namespace:AA.Plugins.Test.ViewModel" xmlns:ns1="clr-namespace:AmmySidekick;assembly=AmmySidekick" xmlns:ns2="clr-namespace:AA.Plugins.Test.ViewModel2" ns1:Ammy.Register="/AA.Plugins.Test;component/Views/AHome.g.xaml">
  <UserControl.Resources>
    <ns0:ViewModelLocator x:Key="PluginLocator"/>
  </UserControl.Resources>
  <UserControl.DataContext>
    <Binding Source="{StaticResource PluginLocator}" Path="Home"/>
  </UserControl.DataContext>
  <TextBox>
    <TextBox.Text>
      <Binding Path="Test123"/>
    </TextBox.Text>
  </TextBox>
</UserControl>`

**Comment by ionoy** [comment]
Hello @danhol86,

Thanks for the feedback!

Is your `ViewModelLocator` defined in `AA.Plugins.Test.ViewModel`? Is it `public`?

**Comment by danhol86** [comment]
Thanks for quick response! Appreciated.

Here is the class its trying to use:

`
namespace AA.Plugins.Test.ViewModel
{
    public class ViewModelLocator
    {
        private static readonly IKernel _kernel;
        static ViewModelLocator()
        {
            _kernel = new StandardKernel();

            if (ViewModelBase.IsInDesignModeStatic)
            {
                _kernel.Bind<IHomeVM>().To<HomeVMDesign>();
                _kernel.Bind<RecordViewVM>().To<RecordViewVM>();
            }
            else
            {
                _kernel.Bind<IHomeVM>().To<HomeVM>().InSingletonScope();
                _kernel.Bind<RecordViewVM>().To<RecordViewVM>().InSingletonScope();
            }
        }

        public IHomeVM Home { get { return _kernel.Get<IHomeVM>(); } }
        public RecordViewVM RecordView { get { return _kernel.Get<RecordViewVM>(); } }
    }
}
`

**Comment by ionoy** [comment]
Seems fine to me. Does Ammy extension show it in red color inside `ammy` file?

**Comment by danhol86** [comment]
Yea. Strange one. The generate XAML doesnt have any problems though. 

These 3 lines are highlighted in red:

Resources: [
	ViewModelLocator Key="PluginLocator" {}
	]

and popup says viewmodellocator does not exist in namespace

**Comment by ionoy** [comment]
And when you rebuild, do you see any warnings?

**Comment by danhol86** [comment]
Ive also noticed that the InitializeComponent(); in the .ammy.cs file is red and says name does not exist in current cotnext?

**Comment by danhol86** [comment]
Have tried cleaning/rebuilding and reopening  etc and still get same error

**Comment by ionoy** [comment]
That's weird. I tried adding the same code to my project and at first, it showed the same error. After I attached to the Visual Studio and tried modifying the `ammy` file the error went away. And now I can't get it to show up again.

**Comment by danhol86** [comment]
Like i say, still works when i run and modify it. Is there any workarounds? So it ignore this error? or just manually type the XAML without it checking it in design mode? Also, if can point me to where this potentially could be occuring, happy to look into a bit more?

**Comment by ionoy** [comment]
The error comes from WPF compiler. But it doesn't show in XAML file, I saw that too. Please try removing this code and pasting it again. Or maybe even retyping it. 
I'm not sure why this error disappeared on my machine, seems like something got cached. Maybe try restarting the Visual Studio?

**Comment by danhol86** [comment]
I have created a new project from scratch just using code mentioned as my existing project may be causing it. It now doesnt show the error direclty in .ammy,  however the same message is shown as warning in Error List tab. Do you get this as well?

**Comment by ionoy** [comment]
Yes, I see this warning in Error List. Not sure what to make of it though.
